packages/Welcome/exported/WelcomeDialog/language-generation/en-us/WelcomeDialog.en-us.lg

```diff
@@ -18,6 +18,6 @@
 ]
 
 # SendActivity_f3uG2y_text()
-- Hi there! I'm here to help you out.
-- Hi, I can help you out today.
-- Hello! I'm ready to help you out today.
\ No newline at end of file
+- Hey, there! How can I help you today?
```

Comment:
The comma seems extraneous (i.e. 'Hey there!' vs 'Hey, there!'). Not sure I've ever seen with a comma?